### DISCUSSION TITLE
Markdown in free plan please??

### DISCUSSION THREAD

**Discussion by mabelslast** [standardnotes/forum#3214] [Suggestions]
It would be nice to at least have formatting with "markdown only" in the free plan and have the formatting buttons in the paid plans. I would not mind manual typing markdown to format my text, at least I can have topics looking like topics and not misplaced text.

Please please please let us use manual formatting with markdown. Love the app, it wins in a list of other feature rich apps, but it's painful to have to read notes with topics/titles that can't be formatted as such 

Please please please give us markdown formatting in the free plan

**Comment by acicovic** [comment]
Today, I was shocked to discover that many note-taking apps which offer E2E sync encryption are actually storing the data on device without any encryption. During my search for an app that immediately encrypts on device, I was delighted to stumble upon Standard Notes.

As a heavy markdown user myself, I think that the request to have manual formatting in the free plan is fair, although I understand that a business needs a business model as well. For me personally, adding that basic markdown support would at least allow me to give an extensive try to the free version in offline use, and then upgrade in the future if I need more features.

**Reply by mabelslast** [reply]
You make some fair points. I hope there's hope.
